content/en/docs/setup/install/single_cluster_eks.md

```diff
@@ -249,6 +252,10 @@ aws s3api create-bucket --bucket "${MILVUS_S3_BUCKET_NAME}" --region "${S3_REGIO
 Pods running in the EKS cluster need to be able to access the S3 bucket. We will create an [IAM role for service account](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) for that.
 
 ``` bash
+export LLMARINER_POLICY="LLMarinerPolicy"
```

Comment:
Not totally necessary, but I prefer to see all of the variables used at the top of bash sections